src/GeositeFramework/Models/Geosite.cs

```diff
@@ -152,7 +153,14 @@ private static List<JsonData> GetPluginConfigurationData(IEnumerable<string> plu
             if (jsonObj["headerLinks"] != null)
             {
                 HeaderLinks = jsonObj["headerLinks"]
-                    .Select(ExtractLinkFromJson).ToList();
+                    .Where(element => element["text"].Value<String>() != "GO TO")
```

Comment:
Ah okay -- happy to do that! I didn't know whether we needed to stick with the existing schema, but I'll modify it and make the corresponding change here.